zaplib/cargo-zaplib/src/build_npm_package.rs

```diff
@@ -0,0 +1,58 @@
+use std::process::Command;
+use log::info;
+
+pub async fn build_npm_package(path: String) {
+  if !std::path::Path::new(&format!("{}/zaplib/web", path)).is_dir() {
+      return;
+  }
+
+  if std::path::Path::new(&format!("{}/zaplib/web/dist/zaplib_runtime.js", path)).exists() {
+      info!("Found zaplib/web/zaplib_runtime.js; skipping build");
+      return;
+  }
+
+  info!("Did not find zaplib/web/zaplib_runtime; building...");
+
+  let npm_status = Command::new("npm")
+                  .arg("--version")
+                  .stdout(std::process::Stdio::null())
+                  .status();
+  if let Err(_) = npm_status {
+      info!("npm not found; skipping build; please install npm");
+      return;
+  }
+  
+  let yarn_status = Command::new("yarn")
+                  .arg("--version")
+                  .stdout(std::process::Stdio::null())
+                  .status();
+  if let Err(_) = yarn_status{
+      info!("yarn not found; installing yarn");
+      let yarn_install_status = Command::new("npm")
+                   .arg("install")
+                   .arg("-g")
+                   .arg("yarn")
+                  .status();
+      if let Err(_) =  yarn_install_status {
+          info!("yarn install failed; skipping build; please install yarn");
```

Comment:
```suggestion
          info!("Installing yarn failed; skipping build; please install yarn");
```

To disambiguate with "running `yarn install`"